src/scr.c

```diff
@@ -1021,17 +1146,26 @@ static int scr_get_params()
   if ((value = scr_param_get("SCR_PREFIX_PURGE")) != NULL) {
     scr_prefix_purge = atoi(value);
   }
+  if (scr_my_rank_world == 0) {
+     scr_dbg(1, "SCR_PREFIX_PURGE=%d ", scr_prefix_purge);
+  }
 
   /* specify whether to use asynchronous flush */
   if ((value = scr_param_get("SCR_FLUSH_ASYNC")) != NULL) {
     scr_flush_async = atoi(value);
   }
+  if (scr_my_rank_world == 0) {
+     scr_dbg(1, "SCR_FLUSH_ASYNC=%d", scr_flush_async);
+  }
 
-  /* Specify whether our flush will be finalized in poststage (currently
-   * only supported with BBAPI). */
+ /* Specify whether our flush will be finalized in poststage (currently
+    only supported with BBAPI). */
```

Comment:
Just another code style thing: for multi-line comments, we align the stars
```suggestion
 /* Specify whether our flush will be finalized in poststage (currently
  * only supported with BBAPI). */
```